src/errors.tsx

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable import/no-anonymous-default-export */
+export default {
```

Comment:
hmm, can you please update your branch with develop? PR for balance fetching should be merged already, i'd expect not to see this in the diff anymore.

Reply:
updated